projects/ui/src/Apis/gg_hooks.ts

```diff
@@ -59,9 +60,11 @@ export function useGetAppDetails(id?: string) {
 export function useListApiKeysForApp(appId: string) {
   return useSwrWithAuth<ApiKey[]>(`/apps/${appId}/api-keys`);
 }
+export function useGetOauthCredentialsForApp(appId: string) {
+  return useSwrWithAuth<OauthCredential>(`/apps/${appId}/oauth-credentials`);
+}
 
 // region Teams
-const TEAMS_SWR_KEY = "teams";
```

Comment:
Not a critique, but why did this come out?  Was it just bloat?

Reply:
Yeah, it wasn't actually being used as a key and was leftover.